app/lib/main.dart

```diff
@@ -55,19 +53,23 @@ Future<void> startApp() async {
     final license = await rootBundle.loadString('google_fonts/LICENSE.txt');
     yield LicenseEntryWithLineBreaks(['google_fonts'], license);
   });
-  runApp(const Effektio());
+  runApp(Effektio());
 }
 
 class Effektio extends StatelessWidget {
-  const Effektio({Key? key}) : super(key: key);
+  Effektio({Key? key}) : super(key: key);
+
+  final routerDelegate = BeamerDelegate(
+    locationBuilder: Routes.getRoutes(),
+  );
 
   @override
   Widget build(BuildContext context) {
     debugInvertOversizedImages = true; // detect non-optimized images
     return Portal(
       child: Themed(
         child: OverlaySupport.global(
-          child: GetMaterialApp(
```

Comment:
I think not having this wrap or replacing would not trigger the cross-signing overlay as its built around ```GetX``` components. Can you try wrapping ```MaterialApp.router``` with parent ```GetMaterial``` or maybe we have to use ```OverlaySupport``` components.

Reply:
I am using same overlay for internet connection handling and it works fine, You can test once and if it doesn't works, we can look into it. Only a minor change would be needed as the MaterialApp.router is wrapped with OverlaySupport.global itself.